src/core/services/utility.spec.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import {isObject} from './utility';
+
+describe('utility', () => {
+
+	describe('isObject', () => {
+
+		it('should return false when passing number', () => {
+			assert.isFalse(isObject(12));
```

Comment:
Should we follow the assert or expect styles only in specs? Or it should be different depending on purposes?